streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -209,7 +210,8 @@ private void setUpMetrics(final ProcessorContext context, final Map<String, Obje
                 (StreamsMetricsImpl) context.metrics(),
                 context.taskId()
             );
-            removeStatisticsFromMetricsRecorder = true;
+            metricsRecorder.startRecording(Duration.ofMinutes(10));
```

Comment:
Here, I set RocksDB's standard interval for its own regular statistics dump. If anybody has a better idea please speak up. In future, we could provide a user configuration to set this.